internal/kafka/command_consumer_group_lag_list_onprem.go

```diff
@@ -0,0 +1,69 @@
+package kafka
+
+import (
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/examples"
+	"github.com/confluentinc/cli/v3/pkg/kafkarest"
+	"github.com/confluentinc/cli/v3/pkg/output"
+)
+
+func (c *consumerCommand) newLagListCommandOnPrem() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:               "list <group>",
+		Short:             "List consumer lags for a Kafka consumer group.",
+		Args:              cobra.ExactArgs(1),
+		RunE:              c.lagListOnPrem,
```

Comment:
```suggestion
		RunE:              c.listOnPrem,
```

Reply:
There's already a `listOnPrem` for `consumerCommand`.

Reply:
This is the style that matches our other commands.
```suggestion
		RunE:              c.groupLagListOnPrem,
```